Lib/test/_test_multiprocessing.py

```diff
@@ -6844,6 +6844,28 @@ def f(x): return x*x
         self.assertEqual("332833500", out.decode('utf-8').strip())
         self.assertFalse(err, msg=err.decode('utf-8'))
 
+    @support.requires_fork()
```

Comment:
We don't need `@support.requires_fork()` here due to the way the multiprocessing tests are structured.

```suggestion
```